app/src/main/java/fr/free/nrw/commons/filepicker/UploadableFile.java

```diff
@@ -124,13 +126,27 @@ private DateTimeWithSource getFileCreatedDateFromCP(Context context) {
     private DateTimeWithSource getDateTimeFromExif() {
         try {
             ExifInterface exif = new ExifInterface(file.getAbsolutePath());
-            @SuppressLint("RestrictedApi") Long dateTime = exif.getDateTime();
+            String dateTimeSubString = exif.getAttribute(ExifInterface.TAG_DATETIME_ORIGINAL);
+
+            dateTimeSubString = "2022:ss:33";
+            String year = dateTimeSubString.substring(0,4);
```

Comment:
I suggest adding a detailed comment explaining why this is implemented this way, so that future developers do not "fix" it in the future. Plus a link to the issue.